hosts/hetzci/vm/casc/pipelines/ghaf-hw-test-manual.groovy

```diff
@@ -38,6 +38,21 @@ properties([
         ]
       ]
     ],
+    $class: 'ChoiceParameter',
+    name: 'TESTAGENT_HOST',
+    choiceType: 'PT_RADIO',
+    description: '''
+      Select the testagent-host. This parameter allows specifying the exact testagent in case Jenkins controller is
+      connected with multiple agents. Can be used together with both IMG_URL and DEVICE_TAG parameters.'''.stripIndent(),
+    script: [
+      $class: 'GroovyScript',
+      script: [
+        classpath: [],
+        sandbox: true,
+        script: "return ['dev','prod','release']"
+      ]
+    ]
+  ],
```

Comment:
Incorrect indentation. You could just copy this section from `prod` or `dev` pipeline.